coxeter/__init__.py

```diff
@@ -13,9 +13,9 @@
 applications such as inertia tensors.
 """
 
-from . import shape_classes, shape_families
+from . import families, shapes
 from .shape_getters import from_gsd_type_shapes
 
-__all__ = ["shape_classes", "from_gsd_type_shapes", "shape_families"]
+__all__ = ["shapes", "from_gsd_type_shapes", "families"]
```

Comment:
Matches import order.
```suggestion
__all__ = ["families", "shapes", "from_gsd_type_shapes"]
```